templates/core/about/builds.html

```diff
@@ -3,77 +3,106 @@
+        </pre>
+        {%- endfilter %}
+        This approach can be helpful if you need dependencies for building the library, but not for building the documentation.
+    </p>
+
+    <p>
+        To configure the compilation environment, you can add <a href="metadata">package metadata</a> to your <code>Cargo.toml</code>, e.g.:
+        {% filter dedent -%}
+        <pre>
+            <code>
+                [package.metadata.docs.rs]
+                rustc-args = ["--cfg", "docsrs"]
+            </code>
+        </pre>
+        {%- endfilter %}
+        Here, the compiler arguments are set so that <code>#[cfg(docsrs)]</code> (not to be confused with <code>#[cfg(doc)]</code>) can be used for conditional compilation.
+        This approach is also useful for setting <a href="https://doc.rust-lang.org/cargo/reference/features.html">features</a>.
+    </p>
+
+    <h4>Testing documentation builds locally</h4>
+    {%- set build_subcommand = docsrs_repo ~ "/blob/master/README.md#build-subcommand" -%}
+    <p>
+        The Docs.rs <a href="{{ build_subcommand | safe }}">README</a> describes how to build
+        unpublished crate documentation locally using the same build environment as the Docs.rs build agent.
+    </p>
+
+    <h3>Diagnosing failed builds</h3>
+
+    <h4>Missing dependencies</h4>
+    <p>
+        Missing dependencies are a common reason for failed builds.
+        Docs.rs dependencies are managed through
+        <a href="https://github.com/rust-lang/crates-build-env">crates-build-env</a>.
+        See <a href="https://forge.rust-lang.org/docs-rs/add-dependencies.html">Forge</a> for how to add a dependency.
+    </p>
+
+    <h4>Write attempt on read-only directories</h4>
+    <p>
+        Most of the sandbox is a read-only file system, including the source directory of your crate and its dependencies.
+        If your <code>build.rs</code> generates files that are relevant for documentation, consider writing to the
+        <a href="https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates">cargo output directory</a>,
+        passed in the environment variable <code>OUT_DIR</code>.
+    </p>
+
+    <h4>Hitting resource limits</h4>
+    <p>
+        All the builds are executed inside a sandbox with limited resources. The current limits are:
+    </p>
+
+    {{ macros::crate_limits(limits=limits) }}
+
+    <p>
+        If your build fails because it these limits, please
+        <a href="{{ docsrs_repo | safe }}/issues/new/choose">open an issue</a>
+        to get them increased for your crate.
+        Since our build agent has finite resources, we have to consider each case individually.
+        However, network access will not be enabled for any crate.
+    </p>
+
+    <h4>Other failures and requesting rebuilds</h4>
+    <p>
+        If you encounter build failures that do not fall under these criteria, please <a href="{{ docsrs_repo | safe }}/issues">file an issue</a>.
```

Comment:
Same comment about using simple language.
```suggestion
        If your crate fails to build for a reason not listed here, please <a href="{{ docsrs_repo | safe }}/issues">file an issue</a>.
```